invenio_previewer/__init__.py

```diff
@@ -79,20 +82,23 @@
 >>> ext_files_rest = InvenioFilesREST(app)
 >>> ext_blueprints = app.register_blueprint(create_blueprint_from_app(app))
 
-The above modules provides the following features to Invenio-Previewer:
+The above modules provide the following features to Invenio-Previewer:
 
-- Invenio-Assets: JavaScript/CSS bundling for interactive previewers.
-- Invenio-Records-UI: Retrieval of record and access control.
-- Invenio-Files-REST: Access to local files (required only if the previewer
-  plugin needs access to the content of a file).
+- `Invenio-Assets <https://invenio-assets.rtfd.io/>`_: JavaScript/CSS bundling
+  for interactive previewers.
+- `Invenio-Records-UI <https://invenio-records-ui.rtfd.io>`_: Retrieval of
+  record and access control.
+- `Invenio-Files-REST <https://invenio-files-rest.rtfd.io/>`_: Access to local
+  files (required only if the previewer plugin needs access to the content of a
+  file).
 
-Lastly, initialize Invenio-Previewer and create an Flask application context:
+Lastly, initialize Invenio-Previewer and create a Flask application context:
```

Comment:
Here we only initialise Invenio-Previewer, pushing the Flask context is in the next code block.